services/modules/slack/tasks.py

```diff
@@ -0,0 +1,28 @@
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.core.exceptions import ObjectDoesNotExist
+from notifications import notify
+
+from .models import SlackUser
+
+
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class SlackTasks:
+    def __init__(self):
+        pass
+
+    @staticmethod
+    def was_invited(user):
+        try:
+            return SlackUser(user).invited != ''
```

Comment:
You're creating an object here, rather than checking the database. Just navigate though the reverse relationship, user.slack.invited.

Comparing a bool and a string. You can just return the bool.